docs/administration/vulnerability-scanning/_index.md

```diff
@@ -3,30 +3,17 @@ title: Vulnerability Scanning
-
-For a brand new installation:
-
-If no scanner is installed, no system default scanner will be set then;
-
-If only one scanner (either Trivy or Clair) is installed, the installed one will become the system default scanner automatically;
-
-If both Trivy and Clair are installed, Trivy will be the system default scanner then.
-
-For upgrades:
+Harbor provides static analysis of vulnerabilities in images through the open source projects [Trivy](https://github.com/aquasecurity/trivy). To be able to use Trivy you must have enabled Trivy when you installed your Harbor instance (by appending installation options `--with-trivy`). For information about installing Harbor with Trivy, see the [Run the Installer Script](../../install-config/run-installer-script.md).
 
 If the upgrading path is from the version that is >=V1.10 to current version (V2.0) and there was an existing system default scanner “ABC” is set in the previous version, that scanner "ABC" will be kept as system default scanner;
 
-Otherwise, Harbor will do the similar operation to the above brand new installation case.
-{{< /important >}}
+You can also connect Harbor to your own instance of Trivy or to other [additional vulnerability scanners](pluggable-scanners.md) through Harbor's embedded interrogation service. These scanners can be configured in the Harbor UI at any time after installation.
 
-You can also connect Harbor to your own instance of Trivy/Clair or to other additional vulnerability scanners through Harbor's embedded interrogation service. These scanners can be configured in the Harbor UI at any time after installation. For the list of additional scanners that are currently supported, see the [Harbor Compatibility List](../../install-config/harbor-compatibility-list.md#scanner-adapters).
+It might be necessary to connect Harbor to other scanners for corporate compliance reasons, or because your organization already uses a particular scanner. Different scanners also use different vulnerability databases, capture different CVE sets, and apply different severity thresholds. By connecting Harbor to more than one vulnerability scanner, you broaden the scope of your protection against vulnerabilities. For the list of additional scanners that are currently supported, see the [Harbor Compatibility List](../../install-config/harbor-compatibility-list.md#scanner-adapters).
 
-It might be necessary to connect Harbor to other scanners for corporate compliance reasons, or because your organization already uses a particular scanner. Different scanners also use different vulnerability databases, capture different CVE sets, and apply different severity thresholds. By connecting Harbor to more than one vulnerability scanner, you broaden the scope of your protection against vulnerabilities.
+{{< important >}}
+Clair was removed as a default scanner in V2.2. You are still able to use Clair for vulnerability scanning by adding it as additional scanner.
```

Comment:
i'd change it to 'by adding it as an external scanner'

Reply:
updated